test-node.bash

```diff
@@ -594,6 +608,12 @@ if $force_init; then
     run_script send-l1 --ethamount 1 --to address_0x0000000000000000000000000000000000000000 --wait
     run_script send-l2 --ethamount 1 --to address_0x0000000000000000000000000000000000000000 --wait
 
+    if $l2_traffic; then
```

Comment:
previously it was also conditioned with `$l3node`; it's just that it makes sense to generate traffic even without L3, right?